test/verify/check-machines

```diff
@@ -3467,6 +3467,42 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
         b.click('#deactivate-network-test_network4-{0}'.format(connectionName)) # Deactivate transient network
         self.waitNetworkRow("test_network4", connectionName, False) # Check it's not present after deactivation
 
+    def testSnapshotRevert(self):
+        b = self.browser
+        m = self.machine
+
+        if m.image in distrosWithoutSnapshot:
+            return
```

Comment:
We should have the negative check as well, in this case that the snapshots tab is missing. With this approach we are notified when an image refresh happends, and we get the required version of libvirt-dbus and we can update the list.

Otherwise, we will probably just forget the list there and not test it.

Reply:
Fixed. I added a check for the snapshots tab not being present. So once some distro will get a new version of libvirt-dbus, it will fail and I will be forced to update the distrosWithoutSnapshot list.
```
        if m.image in distrosWithoutSnapshot:
            b.wait_not_present("#vm-subVmTest1-snapshots")
            return
```